app/models/analyticEvent.server.test.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  getTallyEvents,
+  tallyAnalytics,
+  addAnalyticEvent,
+  getRecentEvents,
+  deleteOldEvents
+} from "./analyticEvent.server";
+
+test("analytic event server: test mocked db part1 tally events", async () => {
+  const intialVal = await getTallyEvents();
+  expect(intialVal).toEqual([]);
+
+  const dateUnixTimeStamps: number[] = [
```

Comment:
can we be more explicit that these are 4 events in dayX and 3 events in dayY?

something like:
dayXTs = 678293369;
dayYTs = 778293369;
dateUnixTimeStamps = [dayXTs, dayXTs+1, dayXTs+2, dayXTs+3, dayYTs, dayYTs+1, dayYTs+2]

Reply:
Resolved.